jumpscale/sals/reservation_chatflow/reservation_chatflow.py

```diff
@@ -1385,6 +1390,19 @@ def get_nodes(
         Returns:
             list of available nodes
         """
+
+        def filter_disallowed_nodes(disallowed_dict, nodes):
```

Comment:
we can make this private method to it's only used here, and to avoid confusion ?

Reply:
it is an inner function anyhow